infra/terraform/templates/launch.tf

```diff
@@ -59,3 +59,8 @@ resource "aws_autoscaling_group" "wellcomecollection_ecs_asg" {
   vpc_zone_identifier       = ["${aws_subnet.public_a.id}", "${aws_subnet.public_b.id}"]
   target_group_arns         = ["${aws_alb_target_group.wellcomecollection.id}"]
 }
+
+output "ami_id" {
+  value = "${data.aws_ami.ecs_optimized.id}"
```

Comment:
P.S. This has always been what we've been using, I just thought the output was useful as it's a looked up variable, so a little blackboxy.